tests/smoke/support/page/files/allFiles.ts

```diff
@@ -95,11 +107,13 @@ export class AllFilesPage {
   async shareFolder({
     folder,
     users,
-    role
+    role,
+    mainMenu
   }: {
     folder: string
     users: User[]
     role: string
+    mainMenu?: boolean
   }): Promise<void> {
```

Comment:
i think it would be nice to rename this param to something like via and then use a enum `... via:QUICK_ACTION ...` then we can reuse this later